File: 32-sum-sq-diff/sum_sq_diff.go
// Golang program to find sum square difference
// Written by M.V.Harish Kumar on 30-05-2024
package main

import (
	"fmt"
	"math"
)

func main() {
	sum_squares := 0
	square_sum := 0

	for i := 1; i <= 100; i++ {
		square_sum += i
		sum_squares += int(math.Pow(float64(i), 2.0))
	}
	square_sum = int(math.Pow(float64(square_sum), 2.0))
	fmt.Println("Sum of squares:", sum_squares)
	fmt.Println("Square of sum:", square_sum)
	fmt.Println("Difference:", square_sum-sum_squares)
}

File: 32-sum-sq-diff/sum_sq_diff.exs
# Elixir program to find sum square difference
# Written by M.V.Harish Kumar on 06-02-2025
sum_squares = 1..100 |> Enum.map(&(&1**2)) |> Enum.sum
square_sum = 1..100 |> Enum.sum |> :math.pow(2)
IO.puts "Sum of squares: #{sum_squares}"
IO.puts "Square of sum: #{square_sum}"
IO.puts "Difference: #{square_sum - sum_squares}"
